msg/BoundingBox.msg

```diff
@@ -1,7 +1,7 @@
 # Note: This message plans to move to geometry_msgs in the final implementation.
 
-# position and rotation of the bounding box
+# position and orientation of the bounding box center
 geometry_msgs/Pose pose
```

Comment:
Not in this PR but related to the other comment. This should probably be more considered an 'origin' or something instead of echoing the datatype.